<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialogDev</class>
 <widget class="QDialog" name="PreferencesDialogDev">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>16</width>
     <height>381</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>280</y>
     <width>351</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Font</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>311</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="1">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QFontComboBox" name="fontComboBox"/>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_font_size"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_font_manager">
       <property name="text">
        <string>Font Manager</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>351</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Colours</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>331</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Incorrect</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_exceptions_color">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Exceptions</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_incorrect_color">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_default_exception_color">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_default_incorrect_color">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>661</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>351</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>User Interface Language</string>
   </property>
   <widget class="QComboBox" name="comboBox_ui_language">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>181</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>English</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Українська</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Deutsh</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>291</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>Languages</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>271</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_browse">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QListWidget" name="listWidget"/>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLineEdit" name="lineEdit_sourceFile"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
